python-modules/cis_profile/cis_profile/profile.py

```diff
@@ -434,7 +434,7 @@ def verify_can_publish(self, attr, attr_name, parent_name=None, previous_attribu
             try:
                 allowed_updators = rules["update"][parent_name][attr_name]
             except TypeError:  # This is not access_information, this is identities or staff_information
-                allowed_updators = rules["create"][parent_name]
+                allowed_updators = rules["update"][parent_name]
```

Comment:
@andrewkrug do you remember what this fixes? the fix looks logical but, i'm wondering how that was found / just in case this impact other code

Reply:
or maybe that was already there and merge shenanigans

Reply:
This wasn't part of my work for this.  This came with the rebase.